service/raft/raft_group0.cc

```diff
@@ -1714,7 +1694,7 @@ void raft_group0::register_metrics() {
     namespace sm = seastar::metrics;
     _metrics.add_group("raft_group0", {
         sm::make_gauge("status", [this] { return static_cast<uint8_t>(_status_for_monitoring); },
-            sm::description("status of the raft group, 0 - disabled, 1 - normal, 2 - aborted"))
+            sm::description("status of the raft group, 0 - normal, 1 - aborted"))
```

Comment:
Hm, this is a backwards incompatible **user-visible** change, we should probably keep `2 - aborted` and say that `1` is unused or sth

Reply:
Yes

Reply:
I think you are right. Also, this `unused` in the description may be understood as if it described the status of the group 0. Group 0 cannot be "unused", so it is misinformation.